src/OAuth/ITokenClient.cs

```diff
@@ -16,5 +16,27 @@ public partial interface ITokenClient
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
         Task<GetAccessTokenResponse> GetAccessTokenFromServicePrincipalAsync(string servicePrincipalKey, AccessKey accessKey, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Gets an access token given a jwt code, client id, redirect URI, client secret and code verifier. These values can be exported from the Laserfiche Developer Console. This is the client credentials flow that applies to service applications.
```

Comment:
Description should be updated. This is for the authorization code flow for web apps and spa